app/_hub/wallarm/wallarm/_index.md

```diff
@@ -76,7 +76,9 @@ More details on [Wallarm API Security Platform](https://www.wallarm.com/product/
 
 Free Wallarm trial license is available [here](https://my.wallarm.com/signup?utm_source=konghub){:target="_blank"}{:rel="noopener noreferrer"}.
 
+"Wallarm implementation was one of many steps to migrate our application iMedNet to AWS. Ensuring the ability to monitor and secure our web-based APIs and to meet HIPAA guidelines for PHI compliance were important in the decision to go with Wallarm”, said Gary Johnson, Infrastructure Architect at MedNetStudy.
```

Comment:
@runscrat I removed these quotes from the previous PR because technical documentation shouldn't contain customer quotes or other marketing type material. If you'd like more of a marketing opportunity for Wallarm's integration with Kong, I'd recommend reaching out to the Kong alliance partners team.

Reply:
@cloudjumpercat , Ok, no problem, thank you!